src/Item/Item.types.ts

```diff
@@ -19,6 +19,7 @@ export type ItemAttributes = {
   id: string // uuid
   /**
    * The urn_suffix field holds the item part of the URN in third party items.
+   * TODO: what about this? The urn_suffix will remain null until the user sets it.
```

Comment:
Should we allow third party items to be created with a null urn_suffix?
These are the two alternatives:
- We could consider that any TWP item upsert MUST have an urn where we will get the suffix from. This implies that the form to create the new items must contain a new field to set the URN.
- We could allow any TPW item upsert to have its URN as null, and let the user set it later.

Reply:
We talked about this a bit, but I think I'd go with option 1. Items MUST have an `urn_suffix`. Maybe we could just pre-fill it with the uuid to avoid having to add a new input to the Modal in the front-end